src/theme/index.js

```diff
@@ -324,8 +323,9 @@ deepmerge(
     },
     MuiIconButton: {
       root: {
+        // Disable ripple effect globally
         borderRadius: 0,
-        "&:hover": {
+        "&:hover, &:focus, &:focus-within": {
           backgroundColor: "inherit",
```

Comment:
Visually I can't see the difference but since we're changing Button hover to transparent, shouldn't we also change here?

Reply:
Ahhh, okay...I'll change that